### PR TITLE
Add language selector to show examples for 4 main langs

### PR DESCRIPTION
**Pull request by Tatskaari**

### CODE REVIEW

**Comment by Tatskaari** [comment]
This is how it looks: 
image

File: docs/example-select.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+function switchTab(event) {
+    let tab = event.target
```
Comment (peterebden) [inline docs/example-select.js:2]: I guess `let` is widely enough supported now that it's fine for  us to use?

File: docs/example-select.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    let lang = tab.dataset.lang
```
Comment (peterebden) [inline docs/example-select.js:4]: semicolons?

**Review by peterebden — APPROVED** [review]

File: docs/example-select.js (hunk 1 above)
Comment (tabazevedo) [inline docs/example-select.js:2]: nit: probably want to go with `const` unless you'll be reassigning these

File: docs/example-select.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    examples.querySelector('.tabs .selected').classList.remove("selected")
+    tab.classList.add("selected")
+
+    examples.querySelector('.examples .selected').classList.remove("selected")
+    examples.querySelector(`.examples > [data-lang="${lang}"]`).classList.add("selected")
+}
+
+document.addEventListener("DOMContentLoaded", function(event) {
+    document.querySelectorAll(".tabs > div").forEach(tab => {
```
Comment (tabazevedo) [inline docs/example-select.js:16]: super nit: probably would want to give the children a `tab` class to make this selector less brittle but i imagine the markup for this won't change very frequently anyway. below you could then also switch to `> .tab` `> .tab:first-child` etc

File: docs/example-select.js (hunk 1 above)
Comment (jakechorleytm) [reply docs/example-select.js:2]: Widely supported at this point https://caniuse.com/?search=let const will be the same

File: docs/example-select.js (hunk 2 above)
Comment (jakechorleytm) [reply docs/example-select.js:4]: Plenty of style guides actively remove them in situations like these

File: docs/example-select.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+function switchTab(event) {
```
Comment (tiagovtristao) [inline docs/example-select.js:1]: This is highly coupled with this use case. I guess it's fine if this isn't supposed to be reused.

Comment (Tatskaari) [reply docs/example-select.js:1]: I suppose i could actually name this stuff as "tabs" and have it be super generic?

File: docs/example-select.js (hunk 3 above)
Comment (Tatskaari) [reply docs/example-select.js:16]: Ah yeah, I guess I'm being lazy.

File: docs/example-select.js (hunk 2 above)
Comment (Tatskaari) [reply docs/example-select.js:4]: Ah we use them in action.js. I'll be consistent.

**Pull request merged**